community/i3-gaps/build

```diff
@@ -0,0 +1 @@
+../i3/build
```

Comment:
This is not a valid `buildfile`.  It should contain the proper build instructions of the application.